### PR TITLE
alpha merge request for greyscale heatmap

### PR DESCRIPTION
**Pull request by imhta**
[Live Preview](https://smashing-magazine-grayscale-heatmap.imhta.now.sh)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,383 @@
...
+      </div>
+      <div class="signin-form">
+        <div class="img-box"></div>
+        <h1><a href="">1,492 smashing members</a></h1>
+        <p>
+          A friendly community that keeps this website alive. With books,
+          webinars, discounts and networking — for just 1 coffee a month. Join
+          the family.
+        </p>
+        <button href="" role="button" class="btn">
```
Comment (bellom) [inline index.html]: Note that `href` is not allowed on `button`, kindly do it in the right way.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,383 @@
...
+      </div>
+      <div class="later-post later-post-5 smash-tv">
+        <div class="img-box"></div>
+        <h1><a href="">New on Smashing TV</a></h1>
+        <p>
+          A channel where seasoned developers show how they design, build, work
+          and play. Coming up next: Video Editing With CSS And JavaScript with
+          Eva Ferreira (Jun 19).
+        </p>
+        <button href="" role="button" class="btn">
```
Comment (bellom) [inline index.html]: Note that `href` is not allowed on element `button`, kindly do it in the right way.

File: index.html [hunk 3]
```diff
@@ -1,13 +1,383 @@
...
+      </div>
+    </section>
+    <section class="conference">
+      <h1>SmashingConf Toronto 2019</h1>
+      <p>
+        Improving accessibility and front-end workflow, refactoring, security,
+        interface design patterns, CSS Grid techniques, performance, Git and how
+        to write better JavaScript
+      </p>
+      <button href="" role="button" class="btn">
```
Comment (bellom) [inline index.html]: Note that `href` is not allowed on element `button`, kindly do it in the right way.

File: index.html [hunk 4]
```diff
@@ -1,13 +1,383 @@
...
+      <div class="img-box"></div>
+      <div>
+      <h1>Design systems</h1>
+      <p>
+        Meet Design Systems, a recent Smashing book in which Alla Kholmatova
+        explores how to set up an effective design system to create great
+        digital products. With common traps, gotchas and lessons learned
+      </p>
+      </div>
+      <button href="" role="button" class="btn">Table of Contents&nbsp;↬</button>
```
Comment (bellom) [inline index.html]: Note that `href` is not allowed on element `button`, kindly do it in the right way.

File: index.html [hunk 5]
```diff
@@ -1,13 +1,383 @@
...
+        <div><a href="" class="nav-item">SMASHING MAGAZINE</a></div>
+        <div><a href="" class="nav-item">ARTICLE</a></div>
+        <div><a href="" class="nav-item">BOOKS</a></div>
+        <div><a href="" class="nav-item">EVENTS</a></div>
+        <div><a href="" class="nav-item">JOBS</a></div>
+        <div><a href="" class="nav-item">MEMBERSHIP</a></div>
+        <div><a href="" class="nav-item">TOPICS</a></div>
+    </nav>
+
+    <div class="latest-articles">
```
Comment (bellom) [inline index.html]: This should be a `main tag` instead of a`div`

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @imhtapm @saintaze, 
**You guys have done a very great job, just few validation errors fixing left and tags issues**
Here is the checklist for your project.
- To consider the project 'complete' it must satisfy the following points:
## General
- [x] The project has a descriptive README.md
## Main Content
The main content should be made from separate containers, stacked on top of each other and in the following arrangement:

* [x] All content should be wrapped with `<main>` HTML tags
* [x] Every children of main should be a `<section>` HTML tag pair
* [x] Every children of section should be an `<article>` HTML tag pair

_After fixing the **unchecked** issues and the validation errors please send another request to re-review this project._
Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

Thanks 🙏

**Comment by maelfosso** [comment]
Hello 

# Good Job!

You can submit the project completion form.

**Pull request merged**